File: center.js
function center (strng, width, fill = ' ') {
    if (strng.length >= width){
      return strng;
    }else{
      let fillAmt = width - strng.length;
    let left = 0;
    let right= 0;
    let word = '';
    if (fillAmt % 2 === 0){
      left = fillAmt/ 2;
      right = fillAmt/ 2;
      word =  fill.repeat(left) + strng + fill.repeat(right);
    }else if (fillAmt % 2 !== 0){
      left = Math.ceil(fillAmt / 2);
      right = Math.floor(fillAmt / 2);
      word =  fill.repeat(left) + strng + fill.repeat(right);
    }
   return word;    
    }  
  }

File: center.py
import math

def center(strng, width, fill=' '):
    if len(strng) >= width:
        return strng
    else:
        left = 0
        right = 0
        word = 0
        fill_amt = width - len(strng)
        if fill_amt %  2 == 0:
            left = fill_amt  // 2
            right = fill_amt // 2
            word = (fill * left) + strng + (fill * right)
            return word
        elif fill_amt % 2 != 0:
            left = math.ceil(fill_amt / 2)
            right = math.floor(fill_amt / 2)
            word = (fill * left ) + strng + (fill * right)
            return word
        